src/OOPExample2/Product.java

```diff
@@ -1,4 +1,4 @@
-package OOPExample2;
+package OOPExample2; //test234
```

Comment:
This is a test! Can you resolve the conversation please? @cristi6891

Reply:
i ll do my best!